CRM/Civixero/Invoice.php

```diff
@@ -184,10 +184,10 @@ public function push($params, $limit = 10) {
         $count++;
       }
       catch (CRM_Civixero_Exception_XeroThrottle $e) {
-        $errors[] = ($this->xero_entity . ' Push aborted due to throttling by Xero');
+        $errors[] = $this->xero_entity . ' Push aborted due to throttling by Xero';
         break;
       }
-      catch (CRM_Core_Exception $e) {
+      catch (Exception $e) {
```

Comment:
Can there be non-`XeroException` messages here? I put handling in `pushToXero()` to convert `XeroException` to `CRM_Core_Exception` - - it also captures the xml - I feel like we should maybe log the xml

Reply:
I couldn't really work out what exceptions would be thrown so I went for the cautious option but if you're happy that it will always be CRM_Core_Exception then we don't need to make this change.